packages/inter-protocol/src/proposals/startEconCommittee.js

```diff
@@ -22,7 +22,7 @@ const sanitizePathSegment = name => {
 
 /**
  * @param {import('./econ-behaviors').EconomyBootstrapPowers} powers
- * @param {object} [config]
+ * @param {object} config
```

Comment:
I know it would be a semantic change, but I would somewhat expect `config` to not be mandatory. Maybe consider changing the runtime args below to provide a default `= {}` value?

Reply:
not worth it imo. all call sites provide the argument.